### ISSUE TITLE
Add Toe hyperphalangy

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10526]
**Preferred term label:**
Toe hyperphalangy

**Synonyms**
-

**Definition (free text, please give PubMed ID)**
Hyperphalangy is a digit morphology in which increased numbers of phalanges are arranged linearly within a toe. That is, there is an accessory phalanx that is arranged linearly with the other phalanges. PMID: 15032905

**Parent term (use hpo.jax.org/app)**
Abnormal toe phalanx morphology HP:0010161

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Desbuquois dysplasia (omim251450), PMC6611735

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000868
- label: Toe hyperphalangy
- parents: [Abnormal toe phalanx morphology (HP:0010161)](https://hpo.jax.org/app/browse/term/HP:0010161)
- pmid: PMID:15032905
- def: Hyperphalangy is a digit morphology in which increased numbers of phalanges are arranged linearly within a toe. That is, there is an accessory phalanx that is arranged linearly with the other phalanges.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10526
- synonyms:
